nixos/doc/manual/release-notes/rl-2505.section.md

```diff
@@ -259,6 +259,8 @@
   to review the new defaults and description of
   [](#opt-services.nextcloud.poolSettings).
 
+- In `users.users` allocation on systems with multiple users it could happen that collided with others. Now these users get new subuid ranges assigned. When this happens, a warning is issued on the first activation. If the subuids were used (e.g. with rootless container managers like podman), please change the ownership of affected files accordingly.
```

Comment:
@leona-ya I feel like a few words are missing in "it could happen that collided with others" – should this say something like "it could happen that <ins>some users' allocated subuid ranges</ins> collided with others"? (I am trying to keep up with release notes and did not at first understand what this was trying to say)

Reply:
Opened https://github.com/NixOS/nixpkgs/pull/408018 – thank you!